server/jobs/job_manager.go

```diff
@@ -77,15 +77,15 @@ func (jm *JobManager) OnConfigurationChange(env mscalendar.Env) error {
 		if enabled && !active {
 			err := jm.activateJob(job)
 			if err != nil {
-				jm.env.Logger.Errorf("Error activating job", "id", job.id, "error", err.Error())
+				jm.env.Logger.Errorf("Error activating %s job. %v", job.id, err)
```

Comment:
Not sure if there is a stronger reasoning for this here, but in some messages you finish the sentence with a dot, and then the error, in some messages you finish the sentence with a colon, and then the error. (0/5) Wouldn't it be better to have them all following the same pattern?